package.json

```diff
@@ -17,31 +17,31 @@
     "yeoman-generator"
   ],
   "dependencies": {
-    "chalk": "^2.4.1",
+    "chalk": "^4.0.0",
     "generator-license": "^5.4.0",
     "lodash": "^4.17.11",
-    "mkdirp": "^0.5.1",
-    "ramda": "^0.25.0",
-    "yeoman-generator": "^3.1.1",
+    "mkdirp": "^1.0.4",
+    "ramda": "^0.27.0",
+    "yeoman-generator": "^4.8.2",
```

Comment:
Hmm... this package really doesn't need the latest of everything, as that makes it rather hard to consume. I'd consider making this just `^4.0.0` as it makes it a lot more consumable then.

- [ ] Don't enforce the latest version of yeoman to be used when it's not needed